app/controllers/games_controller.rb

```diff
@@ -24,6 +25,17 @@ def show
     return render_not_found if @game.blank?
   end
 
+  def update
+    @game = Game.find_by(id: params[:id])
+    return render_not_found if @game.blank?
+    if @game.valid?
+      redirect_to game_path(@game)
+      @game.update_attributes(black_player_id: current_user.id)
```

Comment:
Have the redirect happen after `update_attributes`.

Want it to update and then redirect. 😄